tests/util/skvbc_history_tracker.py

```diff
@@ -26,6 +26,22 @@
 
 MAX_LOOKBACK=10
 
+def verify_linearizability():
+    """
+    Runs the decorated async function for creating tracker
```

Comment:
The doc should rather say:
"Creates a tracker, provides it to the decorated method and checks linearizability of the resulting history."